pkg/agent/datamodel/sig_config.go

```diff
@@ -261,15 +290,17 @@ var AvailableWindowsPIRDistros = []Distro{
 
 // SIG const.
 const (
-	AKSSIGImagePublisher           string = "microsoft-aks"
-	AKSWindowsGalleryName          string = "AKSWindows"
-	AKSWindowsResourceGroup        string = "AKS-Windows"
-	AKSUbuntuGalleryName           string = "AKSUbuntu"
-	AKSUbuntuResourceGroup         string = "AKS-Ubuntu"
-	AKSCBLMarinerGalleryName       string = "AKSCBLMariner"
-	AKSCBLMarinerResourceGroup     string = "AKS-CBLMariner"
-	AKSUbuntuEdgeZoneGalleryName   string = "AKSUbuntuEdgeZone"
-	AKSUbuntuEdgeZoneResourceGroup string = "AKS-Ubuntu-EdgeZone"
+	AKSSIGImagePublisher               string = "microsoft-aks"
+	AKSWindowsGalleryName              string = "AKSWindows"
+	AKSWindowsResourceGroup            string = "AKS-Windows"
+	AKSUbuntuGalleryName               string = "AKSUbuntu"
+	AKSUbuntuResourceGroup             string = "AKS-Ubuntu"
+	AKSAzureLinuxGalleryName           string = "AKSCBLMariner"
+	AKSAzureLinuxResourceGroup         string = "AKS-CBLMariner"
+	AKSAzureLinuxCgroupV2GalleryName   string = "AKSAzureLinux"
```

Comment:
This is because we would like to stop adding new images to the existing SIGs that are named "CBLMariner" once AKS' node imaging service ends for kubernetes version v1.27. And starting from v1.28, we would like to add images to the new SIG named "AzureLinux" only.